sample-data/quotenet-output.csv

```diff
@@ -0,0 +1,43 @@
+﻿TEVA PFD 7,Teva Pharmaceutical Industries Ltd 7 % Conv.Cum.Pfd Shs Mandatory Conv. 15.12.2018,IL0062905489

```

Comment:
Yeah, keep it as short as possible but more importantly as informant and true.
(If you think this length is right, it's fine by me)